tensorflow/core/util/mkl_heuristics.h

```diff
@@ -120,6 +120,71 @@ static double CalculateNodeMFlops(const AttrSlice& attrs,
   return -1;
 }
 
+static bool MatMulHeuristic(const Node* n) {
+  // Run heuristic only if CPU is ARM_NEOVERSE_V1
+  if (!tsl::port::TestAarch64CPU(tsl::port::Aarch64CPU::ARM_NEOVERSE_V1)) {
+    return -1;
```

Comment:
Why are we returning an integer when the function signature returns bool? Please change to `true` and `false`.